README.md

```diff
@@ -2,17 +2,12 @@
 
 The MCP inspector is a developer tool for testing and debugging MCP servers.
 
-It can be run easily from `npx`. For example, in a folder where there's a built JavaScript server at `build/index.js`:
+It can be run easily from `npx`:
 
 ```
-npx @modelcontextprotocol/inspector build/index.js
```

Comment:
The command here is illustrating how to launch the inspector pointed at a built MCP server. So the idea is to run this from something like the `servers` repo as:

```
npx @modelcontextprotocol/inspector src/everything/dist/index.js
```

That will launch the inspector and pre-fill the Command in the UI without you needing to clone this repo at all. We could definitely make this clearer.